MREUnityRuntime/MREUnityRuntimeLib/Core/Physics/JitterBuffer.cs

```diff
@@ -29,7 +30,7 @@ public void Lerp(RigidBodyTransform t0, RigidBodyTransform t1, float f)
 	/// <summary>
 	/// Snapshot of rigid body transforms at specified point in time.
 	/// </summary>
-	public class Snapshot
+	public struct Snapshot
```

Comment:
>struct [](start = 8, length = 6)

why?

Reply:
Ah, not necessary anymore, struct was at one stage needed for non-nullable